main/guides/zoe/contract-basics.md

```diff
@@ -1,46 +1,47 @@
 # Smart Contract Basics
 
 Before we look at how to make a contract such as the one in [the
-basic dapp](../getting-started/) in the previous section, let's cover some basics.
+basic dapp](../getting-started/) in the previous section, let's cover some basics by writing a simple contract that returns a greetings message. We will simply call it _greetings smart contract_. 
 
-A contract is defined by a JavaScript module that exports a `start` function
-that implements the contract's API.
+A contract is defined by a JavaScript module that exports a `start` function. For our greetings smart contract, the declaration of `start` function looks like this:
 
 <<< @/../snippets/zoe/src/01-hello.js#start
 
-Let's start with a contract with a simple `greet` function:
+For the greetings smart contract, we will have a simple `greet` function apart from `start` function. The `greet` function takes a string as a parameter (for example, name of the person calling the function) and returns a customized greeting message.
 
 <<< @/../snippets/zoe/src/01-hello.js#greet
 
-The `start` function can expose the `greet` function
-as part of the contract API by making it
-a method of the contract's `publicFacet`:
+The `greet` function, along with any other public function, must be made accessible through the `publicFacet` of the contract. The `start` function returns an object with a `publicFacet` property. In the greeting contract, the `start` function exposes the `greet` function by defining it as a method of the contract's `publicFacet`, as shown below:
 
 <<< @/../snippets/zoe/src/01-hello.js#publicFacet
 
-We mark it `Far(...)` to allow callers to use it from outside the contract
-and give it a suggestive interface name for debugging.
+We wrap the `greet` function in a `Far(...)` call to safely expose it as a remote object, accessible from outside the contract.
+We also give it a suggestive interface name `Hello` for debugging.
```

Comment:
TECHNICAL:

This phrasing suggests that giving a suggestive name is separate from wrapping with Far.

```suggestion
This also gives it a suggestive interface name `Hello` for debugging.
```

Reply:
Fixed.